### DISCUSSION TITLE
Is there an S3 upload capability or extension?

### DISCUSSION THREAD

**Discussion by ivandir** [django-cms/django-filer#1443] [Q&A]
What's the recommendation here, DIY or is there something else out there to use with Filer?

**Comment by fsbraun** [comment]
I personally use [django-storages](https://django-storages.readthedocs.io/en/latest/index.html) which works nicely. django-filer uses Django's storages and adds an admin interface. So whatever package you choose that provides a storage interface for Django, it will run fine...
